documentation/en/api-v1-unstable-methods.md

```diff
@@ -1241,10 +1241,12 @@ type IndexValidation struct {
 	TipSetKey TipSetKey
```

Comment:
@aarshkshah1992 I think we should remove the `IndexValidation` struct from here document, it's not readable at all in the `preview mode`

Reply:
Removed.